buzzmobile/tests/unit/process/test_bearing.py

```diff
@@ -16,8 +16,9 @@ def test_distance(self):
         assert np.isclose(calculate_directions.get_distance(fix1, fix2), 2517000, rtol=.01)
 
 
-@with_roscore
-class TestBearingNode(unittest.TestCase):
+class TestBearingNode(RosTest):
+    def setUp(self):
+        pass
```

Comment:
no, I can (and will) remove it. I was testing some metaclass errors I was encountering related to the class dict.